cmd/sync.go

```diff
@@ -87,7 +89,8 @@ func newSyncCmd() *cobra.Command {
 	}
 
 	cmd.Flags().StringVar(&syncFromDate, "from-date", "", "Date you want to synchronize charts from. Format: YYYY-MM-DD")
-	cmd.Flags().StringVar(&syncWorkdir, "workdir", syncer.DefaultWorkdir(), "Working directory.")
+	cmd.Flags().StringVar(&syncWorkdir, "workdir", syncer.DefaultWorkdir(), "Working directory")
+	cmd.Flags().BoolVarP(&insecure, "insecure", "", false, "Allow insecure SSL connections")
```

Comment:
If we are not going to use the "shorthand letter" param, why not using "BoolVar" instead of "BoolVarP" (https://github.com/spf13/pflag/blob/master/bool.go#L53)